R/mod_utilization.R

```diff
@@ -0,0 +1,151 @@
+#' utilization UI Function
+#'
+#' @description A shiny Module.
+#'
+#' @param id,input,output,session Internal parameters for {shiny}.
+#'
+#' @noRd 
+#'
+#' @importFrom shiny NS tagList 
+# mod_UI
+mod_utilization_ui <- function(id){
+  ns <- shiny::NS(id)
+  shiny::fluidPage(
+    ui_header_row(),
+    ui_row(box(
+      pickerInput(
+        inputId = "providerListUtilization",
+        choices = c(sort(utilization_bed$ProjectName)),
+        options = pickerOptions(dropupAuto = FALSE,
+                                liveSearch = TRUE),   
+        selected = sample_n(utilization_bed |> select(ProjectName), 1),
+        width = "100%"
+      ),
+      airDatepickerInput(
+        inputId = "utilizationDate",
+        label = "Report End Month",
+        maxDate = ymd(floor_date(update_date, unit = "month") - 1),
+        minDate = ymd(floor_date(update_date - 335, unit = "month")),
+        dateFormat = "MM yyyy",
+        view = "month",
+        value = ymd(floor_date(update_date, unit = "month") - 1),
+        minView = "months",
+        addon = "none",
+        autoClose = TRUE,
+        width = '25%'
+      ),
+      width = 12
+    )), 
+    plotlyOutput("bedPlot"),
+    br(),
+    ui_row(
+      title = "Summary",
+      fluidRow(
```

Comment:
`ui_row` handles adding the row, making this superfluous. Only the columns need be added directly